README.md

```diff
@@ -8,13 +8,23 @@ Paper: http://arxiv.org/abs/2208.01066 <br><br>
 ## Getting started
 You can start by cloning the repo and following the steps below.
 
+
 1. Install the dependencies for our code using Conda. You may need to adjust the environment YAML file depending on your setup. *(we did not)*
 
     ```
     conda env create -f environment.yml
     conda activate in-context-learning
     ```
 
+    If you are trying to run the above lines in GitHub Codespaces and have created the environment.yml file but can't activate the in-context-learning environment, you may have to run the following:
+
+    ```
+    conda init zsh
+    zsh -l
+    ```
+
+    This names the shell to "zsh" then restarts it. Afterwards, you should be able to run "conda activate in-context-learning."
```

Comment:
It adds `/opt/conda/bin/conda` into the path, so we get the activate command -- maybe something like "This adds the necessary binaries to your zsh path and updates your shell. ..."